Subtester/SKConfig.storekit

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
Just to check but this file is included when exporting the project to iOS right?

Reply:
Right now it isn't, once Unity creates the xcworkspace, you need to manually add it. I added logs to the Subtester to remind people of this. 

Ideally we could add it to the xcworkspace that Unity generates, but I wasn't able to find a way to do it - do you know if that's possible?

Reply:
I actually don't know either. I don't think this is a blocker, so I think we can go ahead with this for now 👍